test/gtest/CMakeLists.txt

```diff
@@ -0,0 +1,23 @@
+include(googletest) #include googletest.cmake
+enable_testing()
+include(GoogleTest)
+
+find_package(rocblas)
```

Comment:
Do we need rocblas for testing ? This has already been discovered in a higher level directory

Reply:
@JehandadKhan, seems find_package(rocblas) is needed, otherwise, there will be a Cmake error


[2022-08-03T05:01:39.283Z] CMake Error at test/gtest/CMakeLists.txt:5 (add_executable):

[2022-08-03T05:01:39.283Z]   Target "test_cache" links to target "roc::rocblas" but the target was not

[2022-08-03T05:01:39.283Z]   found.  Perhaps a find_package() call is missing for an IMPORTED target, or

[2022-08-03T05:01:39.283Z]   an ALIAS target is missing?

> Do we need rocblas for testing ? This has already been discovered in a higher level directory

Reply:
@JehandadKhan @xinlipn 
related to this condition rocblas was not imported because it was OpenCL runtime stage.
line from 241 to 280

https://github.com/ROCmSoftwarePlatform/MIOpen/blob/4381ab1ef07b47416b2b9df14e47f6ddf57672e6/CMakeLists.txt#L241

https://github.com/ROCmSoftwarePlatform/MIOpen/blob/4381ab1ef07b47416b2b9df14e47f6ddf57672e6/CMakeLists.txt#L270-L275

The question arises why this test does not handle the flag MIOPEN_USE_ROCBLAS and MIOPEN_BACKEND.
https://github.com/ROCmSoftwarePlatform/MIOpen/blob/4381ab1ef07b47416b2b9df14e47f6ddf57672e6/src/CMakeLists.txt#L676

And I don't see where this test uses rocblas.